torchbench_nvfuser_partitioner.txt

```diff
@@ -0,0 +1,7780 @@
+====== resnext50_32x4d_forward_0 ======
```

Comment:
@jjsjann123 

This is the partition generated for the torch bench models, each line will be a fusion group. 

I will filter out the fusion groups with a single node. 

I see a lot of aten.view ops in the fusion group, does it looks ok to you?

Reply:
We probably don't want to copy the parser support. (those are cpp support but not what python API can handle).